fmriprep/cli/run.py

```diff
@@ -140,6 +140,9 @@ def get_parser():
         '--medial-surface-nan', required=False, action='store_true', default=False,
         help='Replace medial wall values with NaNs on functional GIFTI files. Only '
         'performed for GIFTI files mapped to a freesurfer subject (fsaverage or fsnative).')
+    g_conf.add_argument(
+        '--cifti-output', required=False, action='store_true', default=False,
```

Comment:
What happens if ``--cifti-output`` and ``--fs-no-reconall`` are both set? Is there any error handling here?